app/services/agenda-service.js

```diff
@@ -162,15 +160,6 @@ export default class AgendaService extends Service {
       await submissionActivity.save();
     }
 
-    // load code-list item
-    const defaultDecisionResultCodeUri = isAnnouncement
-      ? CONSTANTS.DECISION_RESULT_CODE_URIS.KENNISNAME
-      : CONSTANTS.DECISION_RESULT_CODE_URIS.GOEDGEKEURD;
-    const decisionResultCode = await this.store.findRecordByUri(
-      'concept',
-      defaultDecisionResultCodeUri
-    );
-
```

Comment:
announcement has to stay `KENNISNAME`
Not setting `GOEDGEKEURD` should be be behind feature flag (maybe, users might want this now? feature flag for now)